### ISSUE TITLE
panels violates rule: aria-hidden-focus

### ISSUE THREAD

**Issue by haonliu** [OfficeDev/office-ui-fabric-react#8911]
## Codepen demonstrating the issue
The first codePen example on the page https://developer.microsoft.com/en-us/fabric#/components/panel

## Actual behavior:
open the panel, run accessibility insights web, aria-hidden-focus rule is reported.

## Expected behavior:
no a11y issue.

## Documentation describing expected behavior
open the panel, run accessibility insights web, aria-hidden-focus rule is reported.

Violation details: 
Title: WCAG 4.1.2: ARIA hidden element must not contain focusable elements (#iFrameKey-68845d4f-6b2a-727b-65f3-77e41be5a7fb;div[aria-hidden="true"][data-is-visible="true"]:nth-child(1))
Tags: Accessibility, WCAG 4.1.2, aria-hidden-focus

Issue: ARIA hidden element must not contain focusable elements (aria-hidden-focus: https://dequeuniversity.com/rules/axe/3.2/aria-hidden-focus?application=msftAI)

Element path: #iFrameKey-68845d4f-6b2a-727b-65f3-77e41be5a7fb;div[aria-hidden="true"][data-is-visible="true"]:nth-child(1)

Snippet: <div tabindex="0" aria-hidden="true" data-is-visible="true" style="pointer-events: none; position: fixed;"></div>

How to fix:
Fix all of the following:
  Focusable content should have tabindex='-1' or be removed from the DOM

Environment:
Chrome version 73.0.3683.103

====

This accessibility issue was found using Accessibility Insights for Web 1.0.4 (axe-core 3.2.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.

**Comment by atneik** [comment]
@haonliu Thanks for reporting this issue. Will you be able to contribute a fix for this one?

**Comment by JasonGore** [comment]
Until we have a browser attribute analogous to `aria-modal` in functionality, I believe an exception will need to be made in Accessibility Insights for cases like these where controls need to approximate the same behavior.
